pkg/discovery/discovery_test.go

```diff
@@ -3,92 +3,100 @@ package discovery
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	. "github.com/go-check/check"
```

Comment:
There is also a `pkg/checker` that defines additional checkers (and *link* the `go-check` ones)